### PR TITLE
Ignore cache and content directories

### PR DESCRIPTION
**Pull request by EBethus**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/cache/*
```
Comment (Kanti) [inline .gitignore]: please alter the Pull Request so git doesn't ignore the ``cache/installedVersion.json`` and ``content/.gitkeep``
than I will merge it.

**Comment by Ashrey** [comment]
Currently those files are not ignored

**Comment by Ashrey** [comment]
Hey, what happened with this push?

**Comment by danpros** [comment]
The `cache/installedVersion.json` and `content/.gitkeep` is needed so please don't ignore it.

**Pull request merged**